### DISCUSSION TITLE
bug

### DISCUSSION THREAD

**Discussion by pandataaz** [FarbstoffRSL/RSL-Helper#352] [Q&A]
dl rsl helper mais le fichiers est introuvable comment faire

**Comment by Morfeo1976** [comment]
![image](https://user-images.githubusercontent.com/105589219/168470395-ec8addf3-0f70-4329-929a-0b41873adbd0.png)
Good mornig,
why?

**Reply by Morfeo1976** [reply]
Infinite loop

**Comment by AndrSilvajr** [comment]
i can't update my RSL Helper .
Info log: Raid or RSLHelper is running. Close or kill the process in taskmanager
and both is not open

**Reply by RedMan1911** [reply]
Kill the task from Taskmanager, but the new version of RSLHelper has not been released yet...

**Reply by AndrSilvajr** [reply]
Don't have ane execution off RSL Helper and RSL in Taskmanager.
I reset the Pc,Don't have ane execution off RSL Helper and RSL in Taskmanager. I reset the Pc, reinstall the program, create a new user and the same error "Raid or RSLHelper is running. Close or kill the process in taskmanager." , and is the firte time i use this program, could you pas me the fonte cod? I looked in Git but is not the cod font. resintal the program, criate a new user.
